samples/choreo-token-exchange-react-app/src/pages/home.tsx

```diff
@@ -0,0 +1,267 @@
+    //             url: `/validate-user`,
+    //             method: "GET",
+    //             shouldAttachIDPAccessToken: true
+    //         });
+    //         setHttpResponse(response?.data);
+    //     } catch (error) {
+    //         setHttpResponse({ status: "Error occured" });
+    //     }
+    // };
+
+    // If `clientID` is not defined in `config.json`, show a UI warning.
+    if (!authConfig?.clientID) {
+        return (
+            <div className="content">
+                <h2>You need to update the Client ID to proceed.</h2>
+                <p>
+                    Please open "src/config.json" file using an editor, and
+                    update the <code>clientID</code> value with the registered
+                    application's client ID.
+                </p>
+                <p>
+                    Visit repo{" "}
+                    <a href="https://github.com/asgardeo/asgardeo-auth-spa-token-exchange/tree/master/samples/asgardeo-react-app">
+                        README
+                    </a>{" "}
+                    for more details.
+                </p>
+            </div>
+        );
+    }
+
+    if (hasLogoutFailureError) {
+        return (
+            <LogoutRequestDenied
+                errorMessage={USER_DENIED_LOGOUT}
+                handleLogin={handleLogin}
+                handleLogout={handleLogout}
+            />
+        );
+    }
+
+    return (
+        <DefaultLayout
+            isLoading={state.isLoading}
+            hasErrors={hasAuthenticationErrors}
+        >
+            {state.isAuthenticated ? (
+                <div className="content">
+                    <AuthenticationResponse
+                        derivedResponse={derivedAuthenticationState}
+                    />
+                    <button
+                        className="btn primary my-4 mx-4"
+                        onClick={() => {
+                            handleLogout();
+                        }}
+                    >
+                        Logout
+                    </button>
+                    <hr className="mt-4" />
+                    {/* <div className="row justify-center">
+                        <button
+                            className="btn primary mx-4"
+                            onClick={() => {
+                                handleRequest();
+                            }}
+                        >
+                            Send Request
+                        </button>
+                        <button
+                            className="btn primary"
+                            onClick={() => {
+                                handleIdpRequest();
+                            }}
+                        >
+                            Send IDP Request
+                        </button>
+                    </div>
+                    {httpResponse && (
+                        <HttpResponse httpResponse={httpResponse} />
+                    )} */}
```

Comment:
Shall we remove this or add a comment mentioning the reason for commenting this out?